cf_xarray/accessor.py

```diff
@@ -1924,9 +1926,10 @@ def get_bounds(self, key: str) -> DataArray:
         DataArray
         """
 
-        return apply_mapper(
-            _variables(_single(_must_exist(_get_bounds))), self._obj, key
-        )[0]
+        results = self.bounds.get(key, [])
+        _raise_if_not_single(key, results)
```

Comment:
Not sure if this is right. For a staggered grid you could have multiple lat/lon with different bounds though I don't know that anyone ever does that :)

Reply:
Mmmm, not sure. I think we would have to change how we pare `da.attrs['bounds']`, because I think right now we expect a single string.

Reply:
Ohh actually you're right, for example key="longitude" could be associated to multiple bounds. The problem is that I think other functions use this and need a single DataArray. Maybe it's better to address this in a separate PR...

Reply:
Lets open an issue for that. And here lets not raise_if_single...